sstmac/hardware/logp/logp_switch.cc

```diff
@@ -92,10 +92,12 @@ LogPSwitch::LogPSwitch(uint32_t cid, SST::Params& params) :
 
   nic_links_.resize(top_->numNodes());
 
+#if SSTMAC_INTEGRATED_SST_CORE
```

Comment:
It's actually not necessary for these conditionals to be here, since they are redundant with conditionals in the init functions (ok, avoids some function calls, but in setup). I put these in to make it really clear that the init functions don't do anything and don't need to get called for standalone build. I found this stuff confusing until I realized that. I guess comments would suffice, but the conditionals are harder to miss.

Reply:
I agree, it probably doesn't matter, but it means that we can't change the behavior of the init functions to also do something in standalone mode without modifying these places.  

I'm not asking you to change it, just making a comment.  With `--with-sst-core` becoming the main method we expect to recommend I think this is fine.